src/foam/u2/grid/HeroGridView/GridViewTest.js

```diff
@@ -0,0 +1,130 @@
+foam.CLASS({
+    name: 'DCWorldMain',
+    extends: 'foam.u2.Element',
+
+    implements: [ 'LoadDCWorld' ],
+
+    requires: [ 'DCWorldController' ],
+
+    properties: [
+        {
+            name: 'controller',
+            factory: function(){
+                return this.DCWorldController.create();
+            }
+        }
+    ],
+
+    methods: [
+
+        function initE(){
+            this.add(this.controller);
+        },
+
+        function init() {
+            this.controller.write();
+        }
+    ]
+});
+
+
+foam.CLASS({
+  name: 'DCWorldController',
+  extends: "foam.u2.Element",
+
+  
+  requires:
+  [
+    'Team',
+    'Hero', 
+    'foam.u2.TableView',
+    'foam.u2.grid.GridView', 
+    ],
+  
+  imports: [
+    'TeamDAO',
+    'HeroDAO', 
+  ], 
+
+  properties: [
+    'gridView', 
+  ],
+
+  methods: [
+    
+    function initE(){
+       this.
+       start(this.STOP, {data: this}).end();
+       this.start("h3").add("Using PropertiesDAO: ").end("h3");
+       this.add(this.gridView);
+       //this.start("h3").add("Using PropertiesArray: ").end("h3");
+       //this.add(this.arrGridView$);
+       }, 
+    
+    function init() {
+        this.gridView = this.GridView.create({
+                        of: this.Hero,
+                        data$: this.HeroDAO$, 
+                        rowProperty: this.Hero.ORGANIZATION, //eq(rowProperty, rowProperties[i])
+                        cellView: 'HeroCellView',
+                        cellWrapperClass: 'NewHeroWrapperView',  
+                        colProperty: this.Hero.STATUS, 
+                        rowPropertiesDAO: this.TeamDAO, // or pass in rowDAO //make it dao based.
+                        matchRowId: true, 
+                        colPropertiesArray: ['alive', 'dead', 'MIA', undefined], //or pass in colDAO
+                        rowDAOMatchUndefined: true,
+                        contextSource: this, 
+                });
+        
+        /*     
+      this.arrGridView = this.GridView.create({
+        data$: this.HeroDAO$,
+        of: this.Hero, 
+        rowProperty: this.Hero.ORGANIZATIONID, //eq(rowProperty, rowProperties[i])
+        colProperty: this.Hero.STATUS, 
+        rowPropertiesArray: ['TS', 'B'], // or pass in rowDAO //make it dao based.
+        colPropertiesArray: ['dead','alive'], //or pass in colDAO
+      });
+      */
+      
+
+    }
+  ],
+  
+      actions:
+    [
+
+        {
+            name: "stop",
+            code: function(){
+                debugger;
```

Comment:
I put the stop button there so when I'm in dev console, I can click stop, and I'd be right where I want in the code. In this case, "this" would be in GridViewTest.
Else, "this" will refer to the window, and I don't know how to find my way to anything.